training/main/scripts/create_kb.py

```
@@ -148,7 +147,7 @@
     return aliases
 
 
-def _load_ents_from_data(nlp: Language, splits: List[str] = ["dev", "train"]):
+def _load_ents_from_data(nlp: Language, splits: list[str] = ["dev", "train"]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)